### PR TITLE
Make so we no longer show contractMarketInfo on memberDetails and pickedLocale not editable

### PR DESCRIPTION
**Pull request by vonElfvin**
# Jira Issue: []

## What?
- Fix member details page
- Add language to member info
- Change name of H.OPE. to Hope
- Add Birthdate celebration support to Hope 😄 
- Allow access to debt tab if no contract is present

## Why?
- Because I brök it
- Language info is nice I guess
- Hope is the future
- Fun thing when it happens
- To be able to whitelist members

## Optional screenshots
image

## Optional checklist
- [x] Updated `src/changelog.ts`
- [x] Codescouted
- [x] Unit tests written
- [x] Tested locally

### CODE REVIEW

File: src/components/member/tabs/DetailsTab.tsx [hunk 1]
```diff
@@ -81,6 +82,7 @@ const DetailsTabComponent: React.FC<WithShowNotification & {
   }
 
   delete memberInfoWithoutSsn.__typename
+  delete memberInfoWithoutSsn.contractMarketInfo
```
Comment (rasmusguterstam) [inline src/components/member/tabs/DetailsTab.tsx:86]: This feels really weird, why are we doing this? And are we really sure we want to do this outside any `useEffect`? 👀

File: src/components/member/tabs/debt-tab/index.tsx [hunk 2]
```diff
@@ -15,37 +14,52 @@ import { MainHeadline } from 'hedvig-ui/typography'
...
         Issue retrieving debt flag for this member
       </StandaloneMessage>
     )
   }
 
+  const eligibleForWhitelist =
+    !person?.status?.whitelisted &&
+    (person?.debt?.paymentDefaults?.length !== 0 ||
+      (person?.debt?.totalAmountDebt.amount ?? 0) > 0)
+
```
Comment (rasmusguterstam) [inline src/components/member/tabs/debt-tab/index.tsx:62]: Maybe make this into a more readable function, with some early returns instead?

File: src/graphql/use-get-quotes.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const useQuotes = (memberId: string): GetQuotesReturnTuple => {
       memberId,
     },
   })
-  const quotes = queryResult.data?.member?.quotes as Quote[] | undefined
+  const quotes = queryResult.data?.member?.quotes
   const contractMarket =
     queryResult.data?.member?.contractMarketInfo ?? undefined
```
Comment (rasmusguterstam) [inline src/graphql/use-get-quotes.tsx:29]:
```suggestion
  const contractMarket =
    queryResult.data?.member?.contractMarketInfo
```

File: src/utils/contract.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ export const getSignSource = (signSource: SignSource): string => {
     case SignSource.Ios:
       return 'iOS'
     case SignSource.Hope:
-      return 'H.OPE.'
```
Comment (rasmusguterstam) [inline src/utils/contract.tsx:12]: nice

**Review by rasmusguterstam — APPROVED** [review]
Really nice, and sweet with some tests! Just some minor comments / questions.

image

File: src/graphql/use-get-quotes.tsx (hunk 3 above)
Comment (vonElfvin) [reply src/graphql/use-get-quotes.tsx:29]: It can be null tho 😢

Comment (rasmusguterstam) [reply src/graphql/use-get-quotes.tsx:29]: oh

File: src/components/member/tabs/DetailsTab.tsx (hunk 1 above)
Comment (vonElfvin) [reply src/components/member/tabs/DetailsTab.tsx:86]: As of now yes, this should go into the bin at some point though after being done properly with graphql

**Pull request merged**